hal/src/rtl872x/usbd_device.cpp

```diff
@@ -332,6 +332,18 @@ int Device::getRawString(const char* data, size_t len, uint8_t* buf, size_t bufl
     return len + 2;
 }
 
+const char * Device::getPlatformUsbName(void) {
```

Comment:
This should not be here, let's move into `deviceid_hal` or into `device_code` and introduce a method to get this, similar to product series.